### ISSUE TITLE
Pull in data/Add instructions for how to request council minutes via FOI requests

### ISSUE THREAD

**Issue by xavriley** [jms301/pfi-explorer#18]
Part of the issue with PFI in general is that all the information is scattering to the wind. The papertrail moves between the stock exchange, central government, local government, nhs trusts and so on until there's no clear picture left.

I came across this https://www.whatdotheyknow.com/request/pfi_sandhill_view_school which is an example of someone successfully requesting minutes from a local council under Freedom of Information law. The levels of bullshit that the requester had to wade through are interesting in their own right. Councils are clearly afraid to release *anything* because the contracts are ongoing citing "commercial sensitivity". The successful arguments used by the requester are very instructive.

I think we should consider a few things in light of this:

  a) Document how citizens should make similar requests for information on other PFI projects. Important to make sure they use WhatDoTheyKnow to keep this information in the open.
  b) Provide some systematic way to link this evidence back to PFI explorer
  c) Orchestrate a way for a user/citizen to "adopt a PFI" and go about retrieving the same information. I think it's best to have a one user -> one PFI project relationship to make sure that we don't end up spamming the relevant public bodies

What do people think? Once we've got a few successful FOI requests under our belt it might be a nice one to get campaigners involved with.

**Comment by xavriley** [comment]
To save people some reading that FOI request refers to this project

http://pfi-explorer.meteor.com/project/120

And the council eventually coughed up these documents

[Report submitted to Council outlining the key legal and financial terms of the Sandhill View PFI project](https://www.whatdotheyknow.com/request/28914/response/87365/attach/html/5/01%2006%2020%20Sandhill%20View%20Part%20II%20Report.doc.html)

[Minutes where the project was approved on the basis of the above report](https://www.whatdotheyknow.com/request/28914/response/88645/attach/html/3/Cabinet%2020%20June%202001.pdf.html)

[Minutes and report outlining the bidding process and the four tendered bids](https://www.whatdotheyknow.com/request/28914/response/87365/attach/html/4/01%2004%2011%20Sandhill%20View%20Part%20II%20Report.doc.html)

In other words, all good stuff!

**Comment by xavriley** [comment]
Another user followed up and got the full 288 page contract

[https://www.whatdotheyknow.com/request/sandhill_view_school_contract](https://www.whatdotheyknow.com/request/sandhill_view_school_contract)

**Comment by xavriley** [comment]
There's a detailed list of Scottish projects here with the dates of the EU tenders http://www.scotland.gov.uk/Resource/0046/00463712.xlsx
